github/resource_github_branch_protection_test.go

```diff
@@ -17,6 +17,7 @@ import (
 func TestAccGithubBranchProtection_basic(t *testing.T) {
 	var protection github.Protection
 
+	resourceName := "github_branch_protection.master"
```

Comment:
Now that we have a variable with the resource name (i might also go with rn over resource name but it doesn't matter) we can replace all instances of that with the variable:
```
					resource.TestCheckResourceAttr(rn, "branch", "master"),
					resource.TestCheckResourceAttr(rn, "enforce_admins", "true"),
					resource.TestCheckResourceAttr(rn, "require_signed_commits", "true"),
					resource.TestCheckResourceAttr(rn, "required_status_checks.0.strict", "true")
```
(applies to all)